indico/modules/categories/templates/display/base.html

```diff
@@ -0,0 +1,123 @@
+<div class="category-container">
+    <div class="category-header">
+        <div id="category-toolbar" class="toolbar right">
+            <div class="group">
+                {% if not category.is_root %}
+                    <a class="i-button icon-arrow-up" href="{{ category.parent.url }}">
+                        {% trans %}Parent category{% endtrans %}
+                    </a>
+                {% endif %}
+                {% if category.event_count %}
+                    <a href="#" id="exportIcal{{ category.id }}"
+                       class="i-button icon-calendar arrow exportIcal"
+                       title="{% trans %}Export to scheduling tool{% endtrans %}"
+                       data-id="{{ category.id }}"></a>
+                    {% with item=category, ics_url=url_for('categories.export_ical', category) -%}
+                        {% include 'categories/category_export_ical.html' %}
+                    {%- endwith %}
+                {% endif %}
+                <a class="i-button icon-eye arrow js-dropdown"
+                   title="{% trans %}View{% endtrans %}"
+                   data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="{# TODO url_for('.today', category) #}">{% trans %}Today's events{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{# TODO url_for('.week', category) #}">{% trans %}Week's events{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{# TODO url_for('.calendar', category) #}">{% trans %}Calendar{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{# TODO url_for('.map', category) #}">{% trans %}Category map{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="{{ url_for('.statistics', category) }}">{% trans %}Category statistics{% endtrans %}</a>
+                    </li>
+                </ul>
+                <a class="i-button icon-plus arrow js-dropdown"
+                   title="{% trans %}Create new event{% endtrans %}"
+                   data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="{{ url_for('event_creation.conferenceCreation', event_type='lecture') }}">
```

Comment:
these links should contain the category id to pre-populate the category selector